.travis.yml

```diff
@@ -13,6 +16,15 @@ env:
 - ARCH=i686
 
 script:
+# prepare qemu
```

Comment:
Could you include a link to Stefan's blog post on qemu in this comment? I assume that's where the line is from right?